src/CSH/Eval/Model.hs

```diff
@@ -0,0 +1,531 @@
+        , ("Project Type: " ++ (show $ projectType p))
+        , ("Project Comments: " ++ (show $ projectComments p))
+        , ("Project Status: " ++ (show $ projectStatus p))
+        ]
+
+data Evaluation = Evaluation {
+    evaluationID           :: Word64
+  , evaluationComments     :: T.Text
+  , evaluationDeadline     :: UTCTime
+  , evaluationAvailable    :: Bool
+  , evaluationStatus       :: EvaluationStatus
+  , evaluationType         :: EvaluationType
+  , evaluationMember       :: Cacheable Member
+  , evaluationConditionals :: Cacheable [Conditional]
+  , evaluationFreshProject :: Cacheable [FreshmanProjectParticipant]
+  }
+
+instance Eq Evaluation where
+    (==) = mkEq evaluationID
+
+instance Show Evaluation where
+    show e = intercalate "\n"
+        [ ("Evaluation ID: " ++ (show $ evaluationID e))
+        , ("Evaluation Comments: " ++ (show $ evaluationComments e))
+        , ("Evaluation Deadline: " ++ (show $ evaluationDeadline e))
+        , ("Evaluation Available: " ++ (show $ evaluationAvailable e))
+        , ("Evaluation Status: " ++ (show $ evaluationStatus e))
+        , ("Evaluation Type: " ++ (show $ evaluationType e))
+        ]
+
+data Conditional = Conditional {
+    conditionalID          :: Word64
+  , conditionalDeadline    :: UTCTime
+  , conditionalDescription :: T.Text
+  , conditionalComments    :: T.Text
+  }
+
+instance Eq Conditional where
+    (==) = mkEq conditionalID
+
+instance Show Conditional where
+    show c = intercalate "\n"
+        [ ("Conditional ID: " ++ (show $ conditionalID c))
+        , ("Conditional Deadline: " ++ (show $ conditionalDeadline c))
+        , ("Conditional Description: " ++ (show $ conditionalDescription c))
+        , ("Conditional Comments: " ++ (show $ conditionalComments c))
+        ]
+
+data FreshmanProject = FreshmanProject {
+    freshmanProjectID           :: Word64
+  , freshmanProjectDescription  :: T.Text
+  , freshmanProjectTerm         :: Cacheable Term
+  , freshmanProjectEvent        :: Cacheable Event
+  , freshmanProjectParticipants :: Cacheable [FreshmanProjectParticipant]
+  }
+
+instance Eq FreshmanProject where
+    (==) = mkEq freshmanProjectID
+
+instance Show FreshmanProject where
+    show f = intercalate "\n"
+        [ ("Freshman Project ID: " ++ (show $ freshmanProjectID f))
+        , ("Freshman Project Description: " ++ (show $ freshmanProjectDescription f))
+        ]
+
+data Packet = Packet {
+    packetID         :: Word64
+  , packetDueDate    :: Day
+  , packetPercentReq :: Integer
+  , packetMember     :: Cacheable Member
+  , packetSignatures :: Cacheable [Signature]
+  }
+
+instance Eq Packet where
+    (==) = mkEq packetID
+
+instance Show Packet where
+    show p = intercalate "\n"
+        [ ("Packet ID: " ++ (show $ packetID p))
+        , ("Packet Due Date: " ++ (show $ packetDueDate p))
+        , ("Packet Percent Required: " ++ (show $ packetPercentReq p))
+        ]
+
+-- This should not be here...
```

Comment:
It should be under "Contexts." Back when we removed the queue table's ID, I neglected to update all the cache code to reflect it. I just noticed tonight.